<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>108</green>
             <blue>21</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>108</green>
             <blue>21</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>160</red>
             <green>162</green>
             <blue>164</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Устный счёт</string>
       </property>
      </widget>
      <widget class="QLabel" name="ans_program">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>761</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="problem">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>781</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>72</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="ans">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>390</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ok">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>390</y>
         <width>88</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_next">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>480</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Далее</string>
       </property>
      </widget>
      <widget class="QLabel" name="important_text">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>791</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
